### PR TITLE
Bug Bounty Scope Suggestion

### PR DESCRIPTION
**Pull request by fapami**
Added some domains I think in scope

### CODE REVIEW

**Comment by antoinet** [comment]
Hi @fapami,
many thanks for your suggestions, highly appreciated. I will need to validate various domains with the corresponding teams, this will probably take a couple of days. Many thanks for your patience.

File: scope/inscope.txt [hunk 1]
```diff
@@ -165,10 +167,14 @@ zabbix.mtf.li
 # Swisscom (Schweiz) AG
 *.bluewin.ch
 *.conextrade.com
+*.curabill.ch
+*.curabill-online.ch
```
Comment (antoinet) [inline scope/inscope.txt]: Service was outphased on 30.06.2022
```suggestion

```

File: scope/inscope.txt [hunk 2]
```diff
@@ -144,9 +144,11 @@
 
 # Global IP Action AG
 *.globalipaction.ch
+*.glipac.ch
```
Comment (byterider) [inline scope/inscope.txt:148]: You could also add *.scsstatic.ch

File: scope/inscope.txt [hunk 3]
```diff
@@ -144,9 +144,11 @@
...
 
 # Innovative Web Marketing & Service AG
 *.i-web.ch
+*.i-web-status.ch
```
Comment (antoinet) [inline scope/inscope.txt]: I haven't received any news from the responsible team yet. Will update as soon as clarified.
```suggestion

```

File: scope/inscope.txt [hunk 4]
```diff
@@ -186,20 +194,29 @@ zabbix.mtf.li
...
 *.mycloud.ch
+*.mytriamed.ch
 *.paybillag.com
 *.pwlan.ch
 *.scbs.ch
 *.sctv.ch
+*.securepop.ch
 *.sinso.ch
 *.smeoffer.ch
+*.sospos.ch
```
Comment (antoinet) [inline scope/inscope.txt]: This belongs to Swisscom Directories AG

File: scope/inscope.txt [hunk 5]
```diff
@@ -186,20 +194,29 @@ zabbix.mtf.li
...
+*.swisscloud.io
 *.swisscom-alarm.ch
 *.swisscom-flaechen.ch
 *.swisscom-health.ch
+*.swisscom-health.it
```
Comment (antoinet) [inline scope/inscope.txt]: Defunct
```suggestion

```

File: scope/inscope.txt [hunk 6]
```diff
@@ -186,20 +194,29 @@ zabbix.mtf.li
...
 *.swisscom-immobilien.ch
 *.swisscom-learningcenter.ch
 *.swisscom-printshop.ch
 *.swisscom-sharespace.com
 *.swisscom-tv.com
+*.swisscomcloud.com
+*.swisscomdata.ch
```
Comment (antoinet) [inline scope/inscope.txt]: Defunct
```suggestion
```

File: scope/inscope.txt [hunk 7]
```diff
@@ -218,8 +235,20 @@ swisscom.ecommerce-benchmark.com
 swisscom.shirthouse.ch
 
 # Swisscom Broadcast AG
+*.bchw.ch
```
Comment (antoinet) [inline scope/inscope.txt]: Defunct
```suggestion
```

File: scope/inscope.txt [hunk 8]
```diff
@@ -218,8 +235,20 @@ swisscom.ecommerce-benchmark.com
...
 *.sbcdc.ch
+*.sbclab.ch
```
Comment (antoinet) [inline scope/inscope.txt]: Defunct
```suggestion
```

File: scope/inscope.txt [hunk 9]
```diff
@@ -218,8 +235,20 @@ swisscom.ecommerce-benchmark.com
...
+*.sbcnms.ch
+*.sbc-nms.ch
+*.sbc-ticketing.ch
+*.sbc-tinsights.ch
```
Comment (antoinet) [inline scope/inscope.txt]: Defunct
```suggestion
```

File: scope/inscope.txt [hunk 10]
```diff
@@ -218,8 +235,20 @@ swisscom.ecommerce-benchmark.com
...
+*.sbcstreams.ch
 *.sbctv.ch
+*.swisscombroadcast.com
+*.swisscombroadcast.ch
+*.swisscom-broadcast-dev.ch
```
Comment (antoinet) [inline scope/inscope.txt]: Defunct
```suggestion
```

File: scope/inscope.txt [hunk 11]
```diff
@@ -275,9 +304,11 @@ swisscom.shirthouse.ch
 *.massagevergleich.ch
 *.metzgereivergleich.ch
 *.multisource.ch
+*.multisourcepro.ch
```
Comment (antoinet) [inline scope/inscope.txt]: Out of scope
```suggestion
```

File: scope/inscope.txt (hunk 2 above)
Comment (antoinet) [reply scope/inscope.txt:148]: scsstatic.ch is defunct

**Review by antoinet — CHANGES_REQUESTED** [review]
Added comments & suggested changes

**Comment by antoinet** [comment]
Thanks for your contributions, @fapami, @byterider

File: scope/inscope.txt (hunk 2 above)
Comment (byterider) [reply scope/inscope.txt:148]: Defunct? Examples: 
https://sdx.scsstatic.ch/
https://rcp.scsstatic.ch/content/dam/assets/b2c/promotionen/apple-iphone-cropped.png

Comment (antoinet) [reply scope/inscope.txt:148]: you are right. will add them to the scope, many thanks!

**Pull request merged**